### DISCUSSION TITLE
Enlarge contact areas (touch display)

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#555] [Q&A] [answered]
When wearing gloves it's sometimes complicated to press the right area on the screen to have an interaction.
see example screenshot

![image](https://github.com/devemux86/cruiser/assets/57968526/c61ac5fe-d270-4f65-a24f-276215f64895)

I want to load a route from my collection for navigation,
Therefore I have to click on the 3 dots and select in the popup menu (not shown here): "load route" So far so good.

Unfortunately the (green) touch area seems rather small (estimated: green marked) so it's almost impossible or tricky to just click
onto the 3 dots without clicking on the "red zone" but clicking on the "red zone" activate or deactivate the data entry and,
when it's activated, in the 3 point menu there are some other functions available but not anymore the "load route"

My proposal(s): Either enlarge the green area that's easier to open for gloves or. Put all functions together into one click - independent if it's marked or not.

Hope it's clear what I mean.

**Comment by devemux86** [comment]
@DIrgi thanks for the suggestion.

We could enlarge the menu touch area.

Also we could show the menu with the long press anywhere on the item?

Currently the long press is to drag the item, like when drag with its icon.
And the tap is to select on / off the item.

---

Note that you can load a route simply with swipe it to the right.

Please see here for the list of the swipe actions in all lists:
- https://github.com/devemux86/cruiser/discussions/469#discussioncomment-8879031

**Reply by DIrgi** [reply]
The "wiping" is a good tip I was not aware about. However doing somehow an enlargement (at least in this case, where we have this small area with the 3 dots) is appreciated as well since wiping always covers only one function.
A "long press" is "ok" as well if this is easier to setup.

Cheers, Dirk

**Reply by MotoUKRider** [reply]
Agree. I also find the 3 dots tricky to handle.
A long-press should behave like a "right-click" and offer a menu like: *"What do you want to do with this item?"*.

Similar to [my recent suggestion](https://github.com/devemux86/cruiser/discussions/540) of the same behaviour on the search results, but there is an apparent limitation with that screen object reacting to a long-press :crying_cat_face:

**Reply by bios4** [reply]
> Also we could show the menu with the long press anywhere on the item?

> A "long press" is "ok" as well if this is easier to setup.

> A long-press should behave like a "right-click" and offer a menu like: "What do you want to do with this item?".

As far as the long-press in these lists currently offers the possibility to rearrange/sort the list's items (long-press --> drag & drop) what would be the solution then to order/rearrange the list when this behaviour is changed (back again)?

And: for the sake of consistency, this changed behaviour needs to be aplied then to ALL lists in Cruiser's screens, where long press/drag currently allows such a rearranging.

**Reply by devemux86** [reply]
Drag to reorder list items can always be done with dragging the item icons at the left.
(simply touch the icon and drag, without long press)

We also allow drag with long press, because long press event is now currently available.

**Reply by bios4** [reply]
> Drag to reorder list items can always be done with dragging the item icons at the left.

HA! I have actually NOT consciously recognized that yet (always thought that reordering is ONLY available by long press/drag)...
Thanks for the hint! 😄

**Comment by DIrgi** [comment]
In general there should be solutions which are most likely intuitive. It does not make sense in general if you need a handbook to remember in all different shortcuts or way of wipings which can be possible for the present function and on the next page another function is different again.

**Comment by devemux86** [comment]
Next Cruiser will also display the menu in all lists with **long press** on the items.

In addition to tap their menu button on the right.

**Comment by devemux86** [comment]
Menu in all lists is now also available with long press on the items:
(in addition to tap their menu button on the right)

- https://github.com/devemux86/cruiser/discussions/610

**Comment by devemux86** [comment, marked as answer]
- Menu in all lists is also available with long press on the items in https://github.com/devemux86/cruiser/discussions/641
(in addition to tap their menu button on the right)
